app/src/pages/playground/PlaygroundInputModeRadioGroup.tsx

```diff
@@ -1,32 +0,0 @@
-import React from "react";
```

Comment:
not needed if we derive state from selection of a dataset - this is the norm for other playgrounds